conf/test.config

```diff
@@ -20,10 +20,6 @@ params {
     max_time   = '6.h'
 
     // Input data
-    // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
-    // TODO nf-core: Give any required params for the test so that command line flags are not needed
-    input  = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_test_illumina_amplicon.csv'
-
-    // Genome references
-    genome = 'R64-1-1'
+    input = "$baseDir/tests/assets/test_samplesheet.csv"
```

Comment:
Could you change this to point to `https://github.com/phac-nml/nf-core-staramr/raw/dev/assets/samplesheet.csv` instead? Two reasons:

1. Using a URL here means it's a bit easier to run without cloning the pipeline (e.g., like `nextflow run phac-nml/nf-core-staramr -profile test,docker ...`
2. Using the file in `assets/samplesheet.csv` means this test profile is in sync with the example samplesheet provided in the git repo. The files under `tests/...` should be preserved for the `nf-test` package in general.

Reply:
Okay! Thanks for the explanation. I'll update this.